sentry-ruby/lib/sentry/transport/http_transport.rb

```diff
@@ -14,6 +14,13 @@ class HTTPTransport < Transport
     RATE_LIMIT_HEADER = "x-sentry-rate-limits"
     USER_AGENT = "sentry-ruby/#{Sentry::VERSION}"
 
+    NET_HTTP_ERRORS = [
```

Comment:
should we maybe use [this list from bundler](https://github.com/ruby/ruby/blob/b0c639f249165d759596f9579fa985cb30533de6/lib/bundler/fetcher.rb#L281-L286)? it has a few more types

Reply:
We can! 

Pros:
- Better coverage! (I didn't think about Zlib errors)

Cons:
- Assumes Bundler is there. 


I think it's an okay trade-off to depend on Bundler?

Reply:
yes just copy over